spec/requests/partners_requests_spec.rb

```diff
@@ -100,7 +100,7 @@
         families_served: 3,
         children_served: 4,
         family_zipcodes: 2,
-        family_zipcodes_list: %w(45612-123 45612-126)
+        family_zipcodes_list: contain_exactly("45612-126", "45612-123") # order of zipcodes not guaranteed
```

Comment:
See here for documentation on `contain_exactly`:

https://rspec.info/features/3-13/rspec-expectations/built-in-matchers/contain-exactly/